static/app/views/replays/detail/issueList.tsx

```diff
@@ -10,174 +10,131 @@ import {DEFAULT_STREAM_GROUP_STATS_PERIOD} from 'sentry/components/stream/group'
-        <IssueDetailsWrapper>
-          <EventOrGroupHeader
-            includeLink
-            data={matchedIssue}
-            organization={organization}
-            size="normal"
-          />
-          <EventOrGroupExtraDetails
-            data={{
-              ...matchedIssue,
-              firstSeen: matchedIssueStats?.firstSeen || '',
-              lastSeen: matchedIssueStats?.lastSeen || '',
-            }}
-          />
-        </IssueDetailsWrapper>
-        {isScreenLarge && (
-          <ChartWrapper>
-            {matchedIssueStats?.stats ? (
-              <GroupChart
-                statsPeriod={DEFAULT_STREAM_GROUP_STATS_PERIOD}
-                data={matchedIssueStats}
-                showSecondaryPoints
-                showMarkLine
-              />
-            ) : (
-              <Placeholder height="44px" />
-            )}
-          </ChartWrapper>
-        )}
-
-        <Item>
-          {matchedIssueStats?.count ? (
-            matchedIssueStats?.count
+function TableRow({
+  isScreenLarge,
+  issue,
+  organization,
+}: {
+  isScreenLarge: boolean;
+  issue: Group;
+  organization: Organization;
+}) {
+  return (
+    <Fragment>
+      <IssueDetailsWrapper>
+        <EventOrGroupHeader
+          includeLink
+          data={issue}
+          organization={organization}
+          size="normal"
+        />
+        <EventOrGroupExtraDetails data={issue} />
+      </IssueDetailsWrapper>
+      {isScreenLarge && (
+        <ChartWrapper>
+          {issue.stats ? (
+            <GroupChart
+              statsPeriod={DEFAULT_STREAM_GROUP_STATS_PERIOD}
+              data={issue}
+              showSecondaryPoints
+              showMarkLine
+            />
           ) : (
-            <Placeholder height="24px" />
+            <Placeholder height="44px" />
           )}
-        </Item>
+        </ChartWrapper>
+      )}
 
-        <Item>
-          {matchedIssueStats?.userCount ? (
-            matchedIssueStats?.userCount
-          ) : (
-            <Placeholder height="24px" />
-          )}
-        </Item>
-      </Fragment>
-    );
-  };
+      <Item>{issue.count ? issue.count : <Placeholder height="24px" />}</Item>
 
-  return (
-    <DiscoverQuery
-      eventView={getEventView()}
-      location={location}
-      orgSlug={organization.slug}
-      limit={15}
-      useEvents
-    >
-      {data => {
-        return (
-          <StyledPanelTable
-            isEmpty={data.tableData?.data.length === 0}
-            emptyMessage={t('No related Issues found.')}
-            isLoading={data.isLoading || loadingIssueData}
-            headers={
-              isScreenLarge ? columns : columns.filter(column => column !== t('Graph'))
-            }
-          >
-            {data.tableData?.data.map(renderTableRow) || null}
-          </StyledPanelTable>
-        );
-      }}
-    </DiscoverQuery>
+      <Item>{issue.userCount ? issue.userCount : <Placeholder height="24px" />}</Item>
```

Comment:
Are these placeholders still needed since you've reduced this down to the 1 request, they should always exist.

Reply:
indeed they are not!